index.js

```diff
@@ -174,8 +221,22 @@ function getOlderCars(/* code here */) {
  * made by either `Audi` or `Mercedes-Benz` or `Volkswagen` or `BMW`,
  * in the same order as they appear in the original inventory.
 */
-function getGermanCars(/* code here */) {
-  /* code here */
+function getGermanCars(arr) {
+  const germanCars = [];
+  
+  for (let i = 0; i < arr.length; i++){
+    switch(arr[i].car_make) {
+      case 'Audi':
+      case 'Mercedes-Benz':
+      case 'Volkswagen':
+      case 'BMW':
+        germanCars.push(arr[i]);
+        break;
+      default:
+        break;
+    }
+  }
+  return germanCars;
 }
```

Comment:
I like that you used a switch statement instead of an if statement because it's easier to read.  I am seeing a few people do this.  I wonder how you guys came up with this.